moirae/estimators/online/filters/conversions.py

```diff
@@ -0,0 +1,191 @@
+                (num_coordinates,)
+
+        Returns:
+            transformed_samples: np.ndarray of the transformed points
+        """
+        raise NotImplementedError('Implement in child class!')
+
+    @abstractmethod
+    def transform_covariance(self, covariance: np.ndarray) -> np.ndarray:
+        """
+        Transforms the covariance of a
+        :class:`~moirae.estimators.online.filters.distributions.MultivariateRandomDistribution`.
+
+        Args:
+            covariance: 2D np.ndarray corresponding to the covariance to be transformed
+
+        Returns:
+            transformed_covariance: transformed covariance matrix
+        """
+        raise NotImplementedError('Implement in child class!')
+
+    @abstractmethod
+    def inverse_transform_samples(self, transformed_samples: np.ndarray) -> np.ndarray:
+        """
+        Performs the inverse tranformation of that given by
+        :meth:`~moirae.estimators.online.filters.transformations.BaseTransform.transform_points`.
+
+        Args:
+            transformed_samples: np.ndarray of points to be inverse transformed
+
+        Returns:
+            samples: np.ndarray corresponding to re-converted points
+        """
+        raise NotImplementedError('Implement in child class!')
+
+    @abstractmethod
+    def inverse_transform_covariance(self, transformed_covariance: np.ndarray) -> np.ndarray:
+        """
+        Performs the inverse tranformation of that given by
+        :meth:`~moirae.estimators.online.filters.transformations.BaseTransform.transform_covariance`.
+
+        Args:
+            transformed_covariance: np.ndarray of transformed covariance to be converted back
+
+        Returns:
+            covariance: np.ndarray corresponding to re-converted covariance
+        """
+        raise NotImplementedError('Implement in child class!')
+
+
+class IdentityConversionOperator(ConversionOperator):
+    """
+    Class that implements simple identity operation, that is, it does not change the inputs.
+    """
+    def transform_samples(self, samples: Optional[np.ndarray]) -> np.ndarray:
+        if samples is None:
+            return
+        return samples.copy()
+
+    def transform_covariance(self, covariance: Optional[np.ndarray]) -> np.ndarray:
+        if covariance is None:
+            return
+        return covariance.copy()
+
+    def inverse_transform_samples(self, transformed_samples: Optional[np.ndarray]) -> np.ndarray:
+        if transformed_samples is None:
+            return
+        return transformed_samples.copy()
+
+    def inverse_transform_covariance(self, transformed_covariance: Optional[np.ndarray]) -> np.ndarray:
+        if transformed_covariance is None:
+            return
+        return transformed_covariance.copy()
+
+
+class LinearConversionOperator(ConversionOperator, validate_assignment=True):
+    """
+    Class that implements a linear function as a transformation (strictly speaking, this is not a linear transformation,
```

Comment:
Could you remove the parenthetical? I think it's too verbose and confusing

Reply:
Sure thing!